### PR TITLE
Finished writing scenarios & implementing login functionality automation

### PR DESCRIPTION
**Pull request by mmahrukh**

### CODE REVIEW

File: src/test/java/com/crm/pages/LoginPage.java [hunk 1]
```diff
@@ -9,10 +9,6 @@
 
 public class LoginPage extends BasePage {
 
-    public LoginPage(){
-        PageFactory.initElements(Driver.getDriver(), this);
-    }
-
```
Comment (StepanRiazhkin) [inline src/test/java/com/crm/pages/LoginPage.java:15]: You don't need this part?

Comment (mmahrukh) [reply src/test/java/com/crm/pages/LoginPage.java:15]: The login page is extending the base page. Through extension, I am already getting that part :)

Comment (StepanRiazhkin) [reply src/test/java/com/crm/pages/LoginPage.java:15]: Ah! What've you done... It's...
.
.
.
.
.
> **E X C E E L L E N T !**

**Review by StepanRiazhkin — APPROVED** [review]
1000% Approved since now!

File: src/test/java/com/crm/pages/LoginPage.java (hunk 1 above)
Comment (mmahrukh) [reply src/test/java/com/crm/pages/LoginPage.java:15]: Haha, Thank you, Thank you :) 

Thanks to Aaron for creating the base page :D

**Pull request merged**